packages/components/src/components/DetailsSearch.vue

```diff
@@ -264,57 +319,81 @@ export default {
   }
 
   &__block {
-    margin-bottom: 1rem;
+    padding: 1.5rem 0.5rem;
+    border-bottom: 2px solid $gray2;
 
     &-title {
       margin: 0 0 0.25rem;
       border-radius: 4px;
       display: inline-block;
-      padding: 0.25rem 0.5rem;
+      padding: 0.25rem 0;
       color: $base01;
-      font-size: 0.75rem;
+      font-size: 0.9rem;
       font-weight: bold;
       text-transform: uppercase;
 
       .details-search__block--http & {
-        background-color: #542168;
+        color: #8e45aa;
       }
 
       .details-search__block--external-service & {
-        background-color: $yellow;
-        color: $base19;
+        color: $yellow;
       }
 
       .details-search__block--labels & {
-        background-color: $base11;
+        color: $base11;
       }
 
       .details-search__block--package & {
-        background-color: $teal;
+        color: $teal;
       }
 
       .details-search__block--class &,
       .details-search__block--function & {
-        background-color: $blue;
+        color: $blue;
       }
 
       .details-search__block--query & {
-        background-color: $royal;
+        color: $royal;
       }
 
       .details-search__block--empty & {
-        background-color: $gray3;
+        color: $gray3;
+      }
+      .details-search__block--analysis-finding & {
+        color: $hotpink;
       }
     }
 
     &-list {
       margin: 0;
-      padding: 0;
+      padding: 0 0.5rem;
       list-style: none;
+      li {
+        &:last-of-type {
+          border-bottom: 0;
```

Comment:
This makes the last label look like it's being clipped
![image](https://user-images.githubusercontent.com/8737782/209198207-17a1382d-9022-4a59-a2d4-aae9461c8eef.png)

Reply:
I'm not sure if this is new or not, but the labels also differ between `DetailsSearch` and `DetailsPanelEvent`
![image](https://user-images.githubusercontent.com/8737782/209200186-eb87a921-e276-4c8e-b527-f140de4125f8.png)